Bottom: 565768ae5ce2e7a88664898f6b81d2470dfde934
Top:    bf3a71d1aaccf85ed8b03f93f13f7bc42bac9fa1
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 15:56:10 +0000

Re-enable soldermask emission


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index efe337c..8d81a8e 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1040,7 +1040,7 @@ object3d_from_soldermask_within_area (POLYAREA *area, int side)
   LayerType *layer;
   POLYAREA *pa;
 
-  return NULL;
+//  return NULL;
 
   poly_Copy0 (&info.poly, area);
   info.side = side;
